signal2html/html.py

```diff
@@ -138,6 +141,41 @@ def format_event_data_group_update(data):
     return event_data
 
 
+def is_empty_message(msg: Dict[str, Any]) -> bool:
+    return not (
+        msg["body"]
+        or (not msg["event_data"] is None)
+        or msg["isCall"]
+        or msg["quote"]
+        or msg["reactions"]
+        or msg["isAllEmoji"]
+        or msg["attachments"]
+    )
+
+
+def filter_date_messages(
+    messages: List[Dict[str, Any]]
+) -> List[Dict[str, Any]]:
+
+    new_messages = []
+    if not messages:
+        return new_messages
+
+    if len(messages) == 1 and messages[0]["date_msg"]:
+        return new_messages
+
+    a, b = iter(messages), iter(messages)
```

Comment:
I think a small comment like
\# Filter consecutive date change messages
would be useful here

Reply:
Good catch! I've added some documentation to the function